src/lambda-powertools-layer.ts

```diff
@@ -122,6 +127,84 @@ export class LambdaPowertoolsLayer extends lambda.LayerVersion {
   }
 }
 
+export class LambdaPowertoolsLayerPythonV3 extends lambda.LayerVersion {
+  /** CDK Layer for Python v3 **/
+  static constructBuildArgs(
```

Comment:
Any reason we have a copy of that? We could extract this as a function and use in both stacks.

Reply:
Good catch! Done